src/app/components/keyboard/keyboard.component.html

```diff
@@ -6,7 +6,7 @@
         *ngFor="let key of firstKeyboardRow"
         (click)="pressKeyValue(key)"
         [ngClass]="newBoxStyles(key)"
-        [disabled]="!this.roundFound || !this.nextRound"
+        [disabled]="!this.roundFound || !this.nextRound || this.round === 5"
```

Comment:
```suggestion
        [disabled]="disableKeyboardRow()"
```
U have 3 times repeated the same checking, extract it to a function and call it from every part